mava/components/jax/building/adders_test.py

```diff
@@ -157,11 +131,15 @@ def test_system_parallel_sequence_adder() -> System:
     return TestSystemWithParallelSequenceAdder()
 
 
+# TODO Fix test.
```

Comment:
Should this first get fixed?

Reply:
I was thinking of doing that in another PR, since this one mainly focuses on the data server tests. I will probably create a new one for the adders.